.travis.yml

```diff
@@ -1,28 +1,12 @@
-language: python
+sudo: required
 
-node_js:
-- 6
-
-python:
-- 2.7
-- 3.5
-- 3.6
-
-before_install:
-- sudo apt-get update
-- sudo apt-get install -qq libzmq3-dev pandoc libcurl4-gnutls-dev libmemcached-dev
-- pip install --upgrade setuptools pip
-- pip install -r requirements-dev.txt
-- npm install .
+language: docker
 
 install:
-- pip install --upgrade setuptools pip
-- pip install -r requirements.txt
-- invoke bower
-- invoke less
+- make build
 
 script:
-- invoke test
+- make test
 
 env:
```

Comment:
I believe the env vars below encode a github key that avoids the tests hitting the rate limit of anonymous calls to the github api. These would need to be forwarded into the container for the tests to use.

Reply:
it captures it from the local environment, rather than having it passed in